src/cmd/windows/durable_task_monitor_test.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+import (
+	"log"
+	"syscall"
+)
+
+func sendSignal(pid int) {
+	d, e := syscall.LoadDLL("kernel32.dll")
+	if e != nil {
+		log.Fatalf("LoadDLL: %v\n", e)
+	}
+	p, e := d.FindProc("GenerateConsoleCtrlEvent")
+	if e != nil {
+		log.Fatalf("FindProc: %v\n", e)
+	}
+	r, _, e := p.Call(syscall.CTRL_C_EVENT, uintptr(pid))
+	if r == 0 {
+		log.Fatalf("GenerateConsoleCtrlEvent: %v\n", e)
+	}
+}
+
+// windows interrupt signals cannot be sent from a program.
+// see: github.com/golang/go/issues/6720
+// IMPORTANT: This test is commented out because it will fail for the following reasons:
+//   1) will ALWAYS fail when executed as RUN command in dockerfile (even on > ltsc2019)
```

Comment:
Wish I could get this test to work. If you run this test inside a custom `ltsc2019` container, it works fine. It's the problem of running it in a dockerfile. Maybe this is not the right way to be doing the tests?